src/Modal/Modal.tsx

```diff
@@ -140,6 +140,11 @@ export const Modal: React.FC<Props> = ({
   primaryAction,
   secondaryAction,
 }) => {
+  const mouseDownTarget = useRef<EventTarget | null>(null);
```

Comment:
would we ever want this to store a reference to something other than the backdrop?

Reply:
nope, are you thinking renaming to something like `backdropMouseDownTarget`?

Reply:
yes please 👍 

I was briefly confused trying to figure out why weren't clearing out the reference in the onClick so the stale one isn't used again

Reply:
(feel free to merge at will following rename btw)